src/core_plugins/console/public/src/directives/history.html

```diff
@@ -9,7 +9,7 @@
       ng-mouseenter="history.viewingReq = req"
       ng-mouseleave="history.viewingReq = history.selectedReq"
       ng-dblclick="history.restore(req)"
-      title="{{ req.description }}"
+      aria-label="{{:: 'Request: ' + history.describeReq(req) }}"
```

Comment:
i did some digging, my notes below for reference.  i'm not going to nit too much, there's room for opinions with the request prefix.

https://github.com/w3c/aria/blob/master/aria/aria.html#L9578:
relevant portions:
`The purpose of aria-labelledby is the same as that of aria-label. It provides the user with a recognizable name of the object`
and 
`If the interface is such that it is not possible to have a visible label on the screen, authors should use aria-label and should not use aria-labelledby`